### DISCUSSION TITLE
Include Non Failures in AWS scan report

### DISCUSSION THREAD

**Discussion by sm171190** [aquasecurity/trivy#4417] [Q&A] [answered]
### Question

Hi everyone,

I see that one of the flags available for the trivy aws command in the Trivy CLI is --include-non-failures. Further the help states _available with '--scanners config'_  However, when I try to run `trivy aws --scanners config --include-non-failures` I get an error 
 `2023-05-17T09:05:35.979+0530	FATAL	unknown flag: --scanners`
The same happens for both the table and JSON output format (I'm interested in JSON btw)
And when I try  `trivy aws  --include-non-failures` I don't get an error but I get the same report as without the  ` --include-non-failures` flag, i.e I don't see any SUCCESS or EXCEPTION in the result.

What would be the correct command to get AWS misconfiguration report with success, failures and exceptions ? 
Thanks
`

### Target

AWS

### Scanner

Misconfiguration

### Output Format

JSON

### Mode

Standalone

### Operating System

Ubuntu

### Version

```bash
Version: dev
Vulnerability DB:
  Version: 2
  UpdatedAt: 2022-04-25 06:06:35.31085973 +0000 UTC
  NextUpdate: 2022-04-25 12:06:35.31085913 +0000 UTC
  DownloadedAt: 2022-04-25 06:42:49.055919668 +0000 UTC
Policy Bundle:
  Digest: sha256:97988c6fc189faa28aa7fc2fc93fb2605c46d2f6ae4092a70e03697ba1d6e620
  DownloadedAt: 2023-05-16 09:24:22.882328508 +0000 UTC
```

**Comment by simar7** [comment]
You're right, this is not supported today. But I'll create an issue to look into it.

**Comment by simar7** [comment, marked as answer]
draft PR here: https://github.com/aquasecurity/trivy/issues/4426
